### ISSUE TITLE
dropoff 5 newscast

### ISSUE THREAD

**Issue by kps501** [kps501/bright_snes9x#3]
This will take a long while. But Bright is mostly complete now.
- maybe add more dsp accuracy fixes (1-4,10)
- add special game hacks
- backport future fixes
- misc libretro fixes

**Comment by kps501** [comment]
@KMFDManic
Before you make a highlight video of Bright, I've got maybe 1 last king card. RE'ing those dsp chips are a pita doughful. And I think the HLE docs are wrong, incorrectly claiming bit perfect.

So I've hooked up byuu's nec dsp code into snes9x Bright. ST010 bios loads and plays 100% so far, since I'm familiar with that one after my research work. And it's the easiest to setup. DSP4 will take some elbow work given how it interfaces. DSP1-3 might be easier to tackle first. ST011 is the outsider oddball.

Super Gameboy won't work - timing is too "critical"; I can't even get ST010 timing working so I kludged it. :)

And I've fixed default Gaussian for Bright (accidental mute output). I'll keep you updated here with any more hotfix preview developments as they're uploaded.

**Comment by KMFDManic** [comment]
@kps501 I will not do anything til you are absolutely in synch with me, communication wise.  I think a July 4th Release Date would be perfect.  Just keep me posted on whatever I need to know, and I will test things accordingly, and post a highlight video, and the release on July 4th!  Top Gear 3000 will be a tremendous perk to this Release, as it is a truly excellent Racing Game!  I will communicate with you, personally, on anything else of pertinence!  It need not be perfect.  Remember, I get a lot of feedback, and it will be very helpful in future commits!  I have already gotten a lot of feedback and communicated with the MAME 2003 Plus Team, and that Core is also becoming better day by day!  Those guys are all great!  Really, everyone I have dealt with from Libretro has been incredibly awesome and helpful and understanding in all that I am trying to achieve:)

**Comment by kps501** [comment]
@KMFDManic 
Preview build now live. System directory: dsp1.bin, dsp2.bin, dsp3.bin, dsp4.bin, st0010.bin

Note that Super Mario Kart is bugged. To use HLE, remove dsp bios atm. Top Gear 3000 looks good all-around.

**Comment by kps501** [comment]
@KMFDManic 
Official build now sent and committed. Fixes dsp1 / st0011 errors. Now all games (minus ST0018,C4) can use hardware emulation. Plus soft reset fixed for internal runahead.

Top Gear 3000 is noticeably improved and F1 ROC II is actually harder to pass around other racers now. SD Gundam GX is finishable. Plus the first Hayazashi Shougi board game works, and will ruthlessly kick your butt.

As a bonus, hardware chipset emulation is fast; it runs faster than SA-1 KDL 3!

Basically I'm done here atm. Thanks for dropping by! Very happy this worked out for both of us. :)
